v0.6.4 (#150)
tendermint/tm-db: CONTRIBUTING.md

# Contributing

> Thank you for considering making contributions to tm-db+!

Reviewer: lol good find

Reviewer: Also I think it should be either
```suggestion
Thank you for considering to make a contribution to tm-db+!
```
or 
```suggestion
Thank you for considering contributing to tm-db+!
```

Reviewer 2: ```suggestion
Thank you for your interest in contributing to tm-db!
```

my 2¢, lol
